x/subspaces/keeper/msg_server.go

```diff
@@ -276,6 +276,11 @@ func (k msgServer) SetUserGroupPermissions(goCtx context.Context, msg *types.Msg
 		return nil, sdkerrors.Wrapf(types.ErrPermissionDenied, "you cannot manage permissions in this subspace")
 	}
 
+	// Make sure that the user is not part of the group they want to change the permissions for
```

Comment:
What if is the subspace owner that wants to change the permissions of the group and is also part of it?

Reply:
Thanks for pointing this out @manu0466. I've updated the code to always allow the owner to set any group permission